front-end/src/IndividualShow.js

```diff
@@ -0,0 +1,188 @@
+import React, { useEffect, useState, useRef } from 'react'
+import Header from './Header'
+import Footer from './Footer'
+import './IndividualShow.css';
+import axios from 'axios';
+import { createMockUser, mockShowAPI, mockUserImage, mockShowImage } from './MockData'
+
+/*the component stores user's watched episode progress 
+by allowing to save the show's latest season and episode watched*/
+const ProgressData = ({ season, episode }) => {
+  const refSeason = useRef();
+  const refEpisode = useRef();
+  const saveProgressData = (() => {
+    //the function will be further developed in sprint 2
+    let progress = [];
+    var season = React.findDOMNode(this.refs.season).value;
```

Comment:
I believe the Professor told us that we should never use var. https://nyu-computer-science.github.io/software-engineering/javascript-intro/#18